channels/alpha

```diff
@@ -17,6 +17,9 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
+  - range: ">=1.7.0"
```

Comment:
Does this relate to this PR (and if so, why :-) )?  I made similar changes to the alpha channel but I don't think we've promoted it to stable yet.